### ISSUE TITLE
Create Communication Channel for translation teams

### ISSUE THREAD

**Issue by cebe** [yiisoft-contrib/yiiframework.com#172]
> 2.0.10 was just released and I didn't have the chance to check/updated any pending messages for the Greek language. There must be a way for any translators to be notified for an imminent release, maybe 3-4 days before the actual release, so they would have the time to update their language's messages. Maybe an open issue that notifies of the imminent release where everyone could subscribe to?
> https://github.com/yiisoft/yii2/pull/12793

We should provide a communication channel for translators, e.g. Forum/Mailinglist/something else? To be able to notify them before a release and also to allow communication between translation team members.

**Comment by lourdas** [comment]
The forum would be nice, but to be honest, I don't visit it very much. I'm sure there are other people like me. So, in addition to the forum, an open issue here in Github would also work, where all translators subscribe to it, so when a new notification message for an imminent release is posted, they would all be notified.

**Comment by bizley** [comment]
I agree. We all are here so GitHub is 100% sure option and the problem with other channels is there is always this one person not using it (Twitter, FB, forum, email, whatever...).

**Comment by cebe** [comment]
Using this channel for now:

We are about to release 2.0.11 soon, so please check your language's translation and update it.

**Comment by githubjeka** [comment]
How about to use [bot](https://github.com/yiisoft-contrib/github-bot) ?

The bot mentions all watchers from `yiisoft/yii2` by `@` in special a issue.

![image](https://cloud.githubusercontent.com/assets/874234/21644602/457f34c0-d29f-11e6-868a-254c95d59a4a.png)

**Comment by cebe** [comment]
that is considered spam by github and not allowed. You can not `@` mention more than a few people in  a message. Not sure what the limit is exactly but it is quite low.

**Comment by arogachev** [comment]
@cebe What about using separate Gitter room? It can be further divided to specific language groups if needed. As far as I remember you was against Gitter because it's a commerical service? But suddenly I saw you today in russian room there. So maybe you have changed your mind?

**Comment by resurtm** [comment]
How is it commercial? They ask money for using rooms there?

**Comment by arogachev** [comment]
@resurtm They planned to add Premium features that are paid, but for the most functionality it's completely free. I don't know, maybe @cebe just prefers IRC based on personal experience or something else.

Google Groups can be another alternative, but personally I'm for Gitter, it's better integrated with Github and pretty functional.

**Comment by resurtm** [comment]
@arogachev 

> They planned to add Premium features that are paid, but for the most functionality it's completely free.

Then I don't get why @cebe thinks it's commercial. By such a logic, Github itself completely commercial too. :-D

**Comment by arogachev** [comment]
@resurtm It was quite a long time ago. When the Gitter came out and Yii2 room appeared I wanted to inform more people to create a bigger community. I wrote to IRC and @cebe responded, I don't remember the exact words, but "commercial" was among them for sure. I'm not offended by that, just don't get it what's wrong with using Gitter. Everyone has his own opinion. 😊

**Comment by jacmoe** [comment]
I'll tell what I felt was wrong with the Yii2 Gitter chat room: it was (probably still is) full of Russians :)

On a serious note: the forum is also an option..

**Comment by samdark** [comment]
@jacmoe because it's linked from the front page of Russian community website.

**Comment by jacmoe** [comment]
Right, that actually makes a lot of sense - I wish the the rest of the Yii community was as strong. :)

**Comment by samdark** [comment]
Posting to community slack channel looks like the best options now.

**Comment by lourdas** [comment]
@samdark What's the Slack channel? I cannot seem to find it...

**Comment by lourdas** [comment]
Found it, sorry for the noise. :)
